src/main/java/ai/zuva/classification/ClassificationRequest.java

```diff
@@ -0,0 +1,124 @@
+        @JsonProperty("file_ids")
+        public ClassificationResult[] results;
+    }
+
+    /**
+     * Send a request to classify the document type of a file.
+     * <p>
+     * Given a ZdaiApiClient and a ZdaiFile, make a request to the Zuva servers to
+     * asynchronously classify the document type of the file. The created ClassificationRequest
+     * object can then be used to query the status and results of the request.
+     *
+     * @param client The client to use to make the request
+     * @param file   The file to classify
+     * @return A ClassificationRequest, which can be used to check the status and results of the request
+     * @throws ZdaiApiException    Unsuccessful response code from server
+     * @throws ZdaiClientException Error preparing, sending or processing the request/response
+     */
+    public static ClassificationRequest createClassificationRequest(ZdaiApiClient client, ZdaiFile file) throws ZdaiClientException, ZdaiApiException {
+        return createClassificationRequests(client, new ZdaiFile[]{file})[0];
+    }
+
+    /**
+     * Send a request to classify the document type of multiple files.
+     * <p>
+     * Given a ZdaiApiClient and an array of ZdaiFiles, make a request to the Zuva servers to
+     * asynchronously classify the document type of the files. The created ClassificationRequests
+     * objects can then be used to query the status and results of each request.
+     *
+     * @param client The client to use to make the request
+     * @param files  The files to classify
+     * @return An array of ClassificationRequests, which can be used to check the status and results of the requests
+     * @throws ZdaiApiException    Unsuccessful response code from server
+     * @throws ZdaiClientException Error preparing, sending or processing the request/response
+     */
+    public static ClassificationRequest[] createClassificationRequests(ZdaiApiClient client, ZdaiFile[] files) throws ZdaiClientException, ZdaiApiException {
+        String[] fileIds = new String[files.length];
+
+        for (int i = 0; i < files.length; i++) {
+            fileIds[i] = files[i].fileId;
+        }
+
+        String body;
+        try {
+            body = client.mapper.writeValueAsString(new ClassificationRequestBody(fileIds));
+        } catch (JsonProcessingException e) {
+            throw (new ZdaiClientException("Error creating request body", e));
+        }
+
+        String response = client.authorizedRequest("POST", "/classification", body, 202);
+
+        try {
+            ClassificationResultsBody resp = client.mapper.readValue(response, ClassificationResultsBody.class);
+            ClassificationRequest[] classificationRequests = new ClassificationRequest[resp.results.length];
+            for (int i = 0; i < classificationRequests.length; i++) {
+                classificationRequests[i] = new ClassificationRequest(client, files[i].fileId, resp.results[i].requestId);
```

Comment:
I think it would be better to get the file ID from the response so that we're not depending on the API returning the requests in the same order that the file ID were submitted.
```suggestion
                classificationRequests[i] = new ClassificationRequest(client, resp.results[i].fileId, resp.results[i].requestId);
```

Reply:
Great catch - I did not think about that at all!